src/node.cc

```diff
@@ -119,6 +119,8 @@ static bool throw_deprecation = false;
 static bool abort_on_uncaught_exception = false;
 static bool trace_sync_io = false;
 static bool track_heap_objects = false;
+static bool profile_cpu = false;
+static const char* profile_title = "";
```

Comment:
If the title is always empty, you might as well drop this variable and use `String::Empty()` below.